index.html

```diff
@@ -0,0 +1,235 @@
+                        <div id="elipse13">
+                            <img src="Ellipse13.png" alt="">
+                        </div>
+                        <div id="elipse15">
+                            <img src="Ellipse15.png" alt="">
+                        </div>
+                        <div id="elipse4">
+                            <img src="Ellipse4.png" alt="">
+                        </div>
+                        <div id="rectangle5">
+                            <img src="Rectangle5.png" alt="">
+                        </div>
+                    </section>
+                </section>
+            </div>
+    </article>
+        
+        <section class="work-card">
+            <div class="work1">
+                <img src="Snapshoot.png" alt="">
+               
+                <h2>Tonic</h2>
+                <div id="frame">
+                    <div id="text1"><h5>CANOPY</h5></div>
+                    <div id="counter"></div>
+                    <div id="role"><h5>Back End Dev</h5></div>
+                    <div id="counter1"></div>
+                    <div id="text"><h5>2015</h5></div>
+                </div>
+                <div class="primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="tags">
+                    <div id="tag1"><div id="html">html</div></div>
+                    <div id="tag2"><div id="css">css</div></div>
+                    <div id="tag3"><div id="javascript">javascript</div></div>
+                </div>
+                <div class="Actions">
+                    <div id="see-project">
+                        <div><h4>See Project</h4></div>
+                    </div>
+                </div>
+            </div>
+            <div class="work2">
+                <img src="SnapshootPortfolio.png" alt="">
+               
+                <h2>Multi-Post Stories</h2>
+                <div id="work2frame">
+                    <div id="work2text1"><h5>CANOPY</h5></div>
+                    <div id="work2counter"></div>
+                    <div id="work2role"><h5>Back End Dev</h5></div>
+                    <div id="work2counter1"></div>
+                    <div id="work2text"><h5>2015</h5></div>
+                </div>
+                <div class="work2primary-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work2tags">
+                    <div id="work2tag1"><div id="html2">html</div></div>
+                    <div id="work2tag2"><div id="css2">css</div></div>
+                    <div id="work2tag3"><div id="javascript2">javascript</div></div>
+                </div>
+                <div class="work2Actions">
+                    <div id="work2see-project">
+                        <div><h4>See Project</h4></div>
```

Comment:
- [ ] As suggested by the previous reviewer,  I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```